sdk/core/core-rest-pipeline/test/retryPolicy.spec.ts

```diff
@@ -94,7 +94,7 @@ describe("retryPolicy", function () {
     });
     await clock.runAllAsync();
     // should be one more than the default retry count
-    assert.strictEqual(next.callCount, 11);
+    assert.strictEqual(next.callCount, 4);
```

Comment:
nit: could you please apply this everywhere applicable?
```suggestion
    assert.strictEqual(next.callCount, DEFAULT_RETRY_POLICY_COUNT  + 1);
```

Reply:
thanks! I had forgotten about this one